Xamarin.MacDev/IAppleSdk.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+
+namespace Xamarin.MacDev {
+	public interface IAppleSdk {
+		bool IsInstalled { get; }
+		string DeveloperRoot { get; }
+		string GetPlatformPath (bool is_simulator);
```

Comment:
minor: iirc fxdg (nor mono) don't suggest underscore in public symbols, including parameters -> pascal case